test/example.js

```diff
@@ -150,6 +150,14 @@ const linksExample = link => console.log('Link:', link);
 const callAsyncFunction = fn =>
   process.nextTick(() => fn(!!Math.floor(Math.random() * 2), 'Yay!'));
 
+// Function with function
+//   fn <Function> function description
+//     num <number> `fn` parameter
+//   Returns: <number>
```

Comment:
Maybe better pad `Returns` to align with arguments
```suggestion
//     Returns: <number>
```

Reply:
We already have comments with such style in https://github.com/metarhia/globalstorage and https://github.com/metarhia/metaschema. Also, in this case, it won't correspond to the level of top-level `Returns:`.